supervisor/dbus/interface.py

```diff
@@ -58,7 +58,7 @@ async def connect(self, bus: MessageBus) -> None:
         await super().connect(bus)
         await self.update()
 
-        if self.sync_properties:
+        if self.sync_properties and self.is_connected:
```

Comment:
Not really related but noticed this in sentry as well. There's a possible race condition where disconnect is called before we get here and then dbus is none. This check should prevent that.